kv/redis/redis.go

```diff
@@ -628,35 +629,56 @@ func (r *Redis) cad(key string, old *store.KeyValue) error {
 		return err
 	}
 
-	return r.runScript(
+	_, err = r.runScript(
 		cmdCAD,
 		key,
 		oldVal,
 	)
+	return err
 }
 
 // Close the store connection
 func (r *Redis) Close() {
 	r.client.Close()
 }
 
+// Increment increments the number stored at key by one
```

Comment:
-> // Increment increases the number stored at key by one

Reply:
OK